ci/test_python.sh

```diff
@@ -31,5 +31,16 @@ rapids-print-env
 rapids-logger "Check GPU usage"
 nvidia-smi
 
+rapids-logger "Checking CLI is available and check discovery is working"
+
+# Sometimes we choose not to explore further.
+# If this is left as a "clean" (non-echo'd) command, the output doesn't get
+# properly grepped
+# shellcheck disable=SC2005
+if ! echo "$(rapids doctor --verbose --dry-run)" | grep -q "Found check"; then
+  echo "Error: 'Found check' not found in output"
+  exit 1
+fi
+
```

Comment:
```suggestion

rapids doctor --verbose
```

Could we try literally running `rapids doctor` (no `--dry-run`) as well, after this?

That'd give you a lot more statement coverage in CI, and be another, independent way to check that the entrypoint registration is working as expected (for example, the text-parsing above confirms that a check was found, but not necessarily that its code is importable).

I'd expect all `rapids doctor` checks to succeed on the CI runners we use to test RAPIDS. If not, then that'd be useful information to have... it might mean we're testing RAPIDS in an environment that this tool would report as not suitable for RAPIDS.